src/Ext/Techno/Hooks.cpp

```diff
@@ -645,3 +645,66 @@ DEFINE_HOOK(0x70265F, TechnoClass_ReceiveDamage_Explodes, 0x6)
 
 	return 0;
 }
+
+DEFINE_HOOK(0x4DEAEE, FootClass_IronCurtain, 0x6)
+{
+	GET(FootClass*, pThis, ESI);
+	GET(TechnoTypeClass*, pType, EAX);
+	GET_STACK(HouseClass*, pSource, STACK_OFFS(0x10, -0x8));
```

Comment:
```suggestion
	GET_STACK(HouseClass*, pSource, STACK_OFFSET(0x10, 0x8));
```
`STACK_OFFS` is deprecated